src/longopt/grdselect_inc.h

```diff
@@ -25,54 +25,33 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'A', "area",
+	          "i,u",               "intersection,union",
+	          "i",                 "increment|inc",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
+	{ 0, 'C', "pointfile",         "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'D', "increment|inc",     "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'E', "tabs",
+	          "b",                 "polygon",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'W', "",
-	          "",                  "",
+	{ 0, 'F', "polygonfile",
 	          "",                  "",
+	          "i,o",               "in,out",
 		  GMT_TP_STANDARD },
-	{ 0, 'Z', "",
+	{ 0, 'G', "force_remote",      "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'I', "invert|reverse",
+	          "C,D,F,L,N,R,W,Z,r", "points,increment,polygons,lines,nans,region,range,zrange,registration",
 	          "",                  "",
+		  GMT_TP_MULTIDIR },
+	{ 0, 'L', "linefile",          "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'M', "margins",           "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'N', "nans",
+	          "l,h",               "lower,higher",
```

Comment:
I would suggest using here "lower|lower_than,higher|higher_than"

Reply:
I am find with those aliases, make it so @rbdavis